deployment/nixops/prometheus.nix

```diff
@@ -180,5 +191,52 @@ in
         ];
       };
     };
+
+    systemd.services.deploymentServer = {
+      enable = enableGithubHooks;
+      path = ["${deploymentServer}" pkgs.git pkgs.nixops pkgs.nix pkgs.gnutar pkgs.gzip ];
+      script = ''deployment-server-exe \
+      --slackChannel ${slackChannel} \
+      --keyfile ${configDir}/secrets.json \
+      --port 8080 \
+      --configDir ${configDir} \
+      --stateFile ${nixopsStateFile} \
+      --deploymentName ${deploymentName} \
+      --include nixos=${nixosLocation} \
+      --include nixpkgs=${nixpkgsLocation}
+      '';
+    };
+
+    services.nginx = {
+      enable = true;
+      statusPage = true;
+
+      recommendedGzipSettings = true;
+      recommendedProxySettings = true;
+      recommendedOptimisation = true;
+
+      upstreams.grafana.servers."127.0.0.1:3000" = {};
+      upstreams.githubwebhooks.servers."127.0.0.1:8080" = {};
```

Comment:
Pull out a constant for the deployment server port.

Reply:
in `let` block now